### PR TITLE
Fix for invalid motion and privacy ui json values (Issue #2423)

### LINKED ISSUE #2423: Wrong mask setting file read for privacy mask ui values
If you enable both motion and privacy mask at the same time, json response for UI will contain motion mask values for both masks(motion and private). Now if you save camera settings both masks will contain same mask(motion area).

Invalid filename forming in **motioneye/utils/__init__.py** function **parse_editable_mask_file**:

```
    file_name = os.path.join(settings.CONF_PATH, 'mask_%s.pgm' % camera_id)
```

Fix:

```
    postfix = "_privacy" if mask_class == "privacy" else ""
    file_name = os.path.join(settings.CONF_PATH, f'mask_{camera_id}{postfix}.pgm')
```

### PR DESCRIPTION
**Pull request by tunkio**
Fix #2423

### CODE REVIEW

File: motioneye/utils/__init__.py [hunk 1]
```diff
@@ -531,7 +531,8 @@ def parse_editable_mask_file(
     # of the camera image, as it might be different from that of the associated mask;
     # they can be null (e.g. netcams)
 
-    file_name = os.path.join(settings.CONF_PATH, 'mask_%s.pgm' % camera_id)
+    postfix = "_privacy" if mask_class == "privacy" else ""
+    file_name = os.path.join(settings.CONF_PATH, f'mask_{camera_id}{postfix}.pgm')
```
Comment (zagrim) [inline motioneye/utils/__init__.py:535]: Seems like the original implementation (which likely went broken when utils.py was refactored for Python3 support) used also here the `build_mask_file_name` function which is still also being used by `build_editable_mask_file` (line 500). Any reason for not using the same file name builder here also?

Comment (zagrim) [reply motioneye/utils/__init__.py:535]: For reference, here are the changes of the original PR for privacy mask support: https://github.com/motioneye-project/motioneye/pull/1247/files#diff-909205a34627609f47cf1071d96dd8ad03166f0617e4af6390faa7bb50034c61

Comment (tunkio) [reply motioneye/utils/__init__.py:535]: No reason at all. Need to change that.

Comment (tunkio) [reply motioneye/utils/__init__.py]: Done

**Review by zagrim — APPROVED** [review]
Tested this locally, seems to work ok.

**Pull request merged**